StrataToCBMC.lean

```diff
@@ -0,0 +1,859 @@
+/-
+  Copyright Strata Contributors
+
+  SPDX-License-Identifier: Apache-2.0 OR MIT
+-/
+
+import Lean.Data.Json
+import Strata.DL.Util.Map
+import Strata.Languages.C_Simp.C_Simp
+import Strata.Languages.C_Simp.Verify
+
+open Lean
+
+-- Our test program
+def SimpleTestEnv :=
+#strata
+program C_Simp;
+
+int procedure simpleTest (x: int, y: int)
+  //@pre y > 0;
+  //@post true;
+{
+  var z : int;
+  //@assume [test_assume_y_pos] y > 0;
+  //@assume [test_assume_x_pos] x > 0;
+  //@assume [test_assume_y_smol] y < 90;
+  //@assume [test_assume_x_smol] x < 90;
+  z = x + y;
+  //@assert [test_assert] z > x;
+  if (z > 10) {
+    z = z - 1;
+  } else {
+    z = z + 1;
+  }
+  return 0;
+}
+
+#end
+
+open Strata.C_Simp in
+def SimpleTestEnvAST := TransM.run (translateProgram (SimpleTestEnv.commands))
+
+def myFunc : Strata.C_Simp.Function := SimpleTestEnvAST.fst.funcs.head!
+
+-- Configuration for CBMC output
+structure CBMCConfig where
+  sourceFile : String := "from_andrew.c"
+  workingDirectory : String := "/home/ub-backup/tautschn/cbmc-github.git"
```

Comment:
I'd like to avoid having code checked into the repo with hard-coded paths that are specific to particular developers.